content/en/blog/2023/any-metric-receiver.md

```diff
@@ -0,0 +1,480 @@
+---
+title: Receive any custom metric with the OpenTelemetry Collector
+linkTitle: Any Metric Receiver
+date: 2023-11-28
+author: '[Severinq Neumann](https://github.com/svrnm), Cisco'
+# prettier-ignore
+cspell:ignore: openssl servername noout enddate gomod debugexporter otlpexporter otlphttpexporter transformprocessor otlpreceiver carbonreceiver ottl datapoint webserver Helmuth
+---
+
+While OpenTelemetry (OTel) is here to help you with troubleshooting and handling
+the _"unknown unknowns"_, it is also instrumental for managing route tasks like
+monitoring system metrics, like disk usage, server availability or SSL
+certificate expiration dates. This can be achieved by utilizing any one of the
+[90+ receivers](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver)
+available for the [OpenTelemetry Collector](/docs/collector), such as the
+[Host Metrics Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver)
+or the [HTTP Check Receiver](/blog/2023/synthetic-testing/).
+
+But what if the available receivers don't meet your specific needs? Suppose you
+have a collection of shell scripts that provide custom metrics, and you want to
+export these to the OpenTelemetry Collector. You could write your own receiver,
+but this would require proficiency in Go programming language and potentially
+converting your scripts to Go.
```

Comment:
Either "in Go" or "in the Go programming language". Given our audience, I'd vote for the former, and even shorten it to the following:

```suggestion
export these to the OpenTelemetry Collector. You could write your own receiver,
but this would require proficiency in Go.
```